### DISCUSSION TITLE
Are there any videos or books to study minio?

### DISCUSSION THREAD

**Discussion by httaotao** [minio/minio#12928] [Q&A] [answered]
Hi, I am gonna  study minio these day, But I can't find any books or videos about minio. so except for the doc, do you have a plan to make some videos to share how to study minio code , like glusterfs [developer-sessions](https://github.com/gluster/glusterfs/tree/devel/doc/developer-sessions) , thanks

**Comment by harshavardhana** [comment, marked as answer]
https://www.youtube.com/results?search_query=minio - just search for MinIO on youtube.

**Comment by guettli** [comment]
I would love to read a book about minio.
